provider_shopper/analysis_options.yaml

```diff
@@ -7,7 +7,6 @@ analyzer:
 
 linter:
   rules:
-    - avoid_types_on_closure_parameters
```

Comment:
We generally try to keep `analysis_options.yaml` files consistent for all the samples. Do we need to remove this lint?

Reply:
Ah, this is a left over change and isn't actually needed. I re-added it.